UIKitHomework/UIKitHomework/FoodViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FoodViewController.swift
+//  UIKitHomework
+//
+//  Created by Vera Zaitseva on 30.06.2021.
+//
+
+import UIKit
+
+class FoodViewController: UIViewController {
+    let pizzaButton = UIButton()
+    let sushiButton = UIButton()
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "Food"
+        self.view.backgroundColor = .white
+        self.navigationItem.setHidesBackButton(true, animated: false) // убрала кнопку назад
+        
+        // Pizza и переход на след экран
+        pizzaButton.frame = CGRect(x: 30, y: 100, width: 350, height: 200)
+        pizzaButton.backgroundColor = .systemPink
+        pizzaButton.setTitle("Пицца", for: .normal)
+        pizzaButton.titleLabel?.font = UIFont.boldSystemFont(ofSize: 20)
+        pizzaButton.setTitleColor(UIColor.white, for: .normal)
+        pizzaButton.layer.cornerRadius = 20
+        self.view.addSubview(pizzaButton)
+        pizzaButton.addTarget(self, action: #selector(goToPizzaController(paramSender:)), for: .touchUpInside)
+        
+        
+        
+        
+        
+        // Суши без перехода
+        sushiButton.frame = CGRect(x: 30, y: 400, width: 350, height: 200)
+        sushiButton.backgroundColor = .lightGray
+        sushiButton.setTitle("Суши", for: .normal)
+        sushiButton.titleLabel?.font = UIFont.boldSystemFont(ofSize: 20)
+        sushiButton.setTitleColor(UIColor.white, for: .normal)
+        sushiButton.layer.cornerRadius = 20
+        self.view.addSubview(sushiButton)
+        
+        
+    }
+    @objc func goToPizzaController(paramSender: Any) {
```

Comment:
```suggestion
    @objc private func goToPizzaController(paramSender: Any) {
```
все методы, которые не вызываются вне скоупа класса должны быть private